lib/go-tc/servers.go

```diff
@@ -87,8 +87,33 @@ type ServerDetailV30 struct {
 
 // ServerDetailV40 is the details for a server for API v4.
 type ServerDetailV40 struct {
-	ServerDetail
-	ServerInterfaces []ServerInterfaceInfoV40 `json:"interfaces"`
+	CacheGroup         *string                  `json:"cachegroup" db:"cachegroup"`
+	CDNName            *string                  `json:"cdnName" db:"cdn_name"`
+	DeliveryServiceIDs []int64                  `json:"deliveryservices,omitempty"`
+	DomainName         *string                  `json:"domainName" db:"domain_name"`
+	GUID               *string                  `json:"guid" db:"guid"`
+	HardwareInfo       map[string]string        `json:"hardwareInfo"`
```

Comment:
Sure.

Reply:
actually that complicates things. I was only always seeing null because my TO instance isn't years old. If you created HWInfo structures back in APIv1, then APIv3's `servers/details` was still showing them. So that's probably something we'd need to take to the mailing list. For all I know that's the entire reason we still have this route.

So probably don't worry about that in this PR.

Reply:
I see.. thanks for the explanation.